apps/site/components/Downloads/ReleaseModal/index.tsx

```diff
@@ -0,0 +1,61 @@
+import Modal from '@node-core/ui-components/Common/Modal';
+import { useTranslations } from 'next-intl';
+import type { FC } from 'react';
+
+import LinkWithArrow from '@/components/LinkWithArrow';
+import { BASE_CHANGELOG_URL } from '@/next.constants.mjs';
+import type { NodeRelease } from '@/types';
+import { getNodeApiLink } from '@/util/getNodeApiLink';
+
+import { MinorReleasesTable } from './MinorReleasesTable';
+
+type ReleaseModalProps = {
+  isOpen: boolean;
+  closeModal: () => void;
+  release: NodeRelease;
+};
+
+const ReleaseModal: FC<ReleaseModalProps> = ({
+  isOpen,
+  closeModal,
+  release,
+}) => {
+  const t = useTranslations('components.releaseModal');
+
+  return (
+    <Modal
+      heading={`Node.js ${release.major} Release`}
+      subheading={release?.codename ?? ''}
+      open={isOpen}
+      onOpenChange={closeModal}
+    >
+      <div className="flex flex-row gap-2">
+        {release.status !== 'Current' && (
+          <LinkWithArrow
```

Comment:
Couldn't we use buttons here, because I find that the modal has a lot of icon arrows?
 
<img width="1440" src="https://github.com/user-attachments/assets/ea6b3ff0-e9c4-4cb8-9857-000d4585fb12" />